.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+**1. Task Description:**
+
+- describe the task if necessary
+
+**2. How to test:**
+
+- `yarn test:<any>` - choose the necessary spec
+
+**3. Comments:**
+
+- to write a comment, if necessary, for a better understanding
+
+**4. Checklist review:**
+
+- [ ] Name of the branch follows the pattern:  title-of-feature
+- [ ] Name of the PR follows the pattern: [FEATURE] title-of-feature
+- [ ] The PR is signed by someone
+- [ ] The PR has the correct label
+- [ ] The build is green
+- [ ] The branch can be deleted
```

Comment:
@uLucasFraga I added some suggestions and removed some unnecessary things (since I revisited the repository settings).

Also, could you please update the file name to be `PULL_REQUEST_TEMPLATE.md`? Just to keep consistency with other GH files

```suggestion
### Description 

Describe the changes from the Pull Request.

### How to test

- `yarn install`
- FIXME: Add steps describing how to test your changes manually

### Checklist

- [ ] Branch name follows the pattern:  `feature-description`
- [ ] PR branch is up-to-date with `main` (if not, rebase it)
- [ ] Double-check the quality of [commit messages](http://chris.beams.io/posts/git-commit/)
- [ ] PR has a descriptive name
```

Reply:
amazing!
sure, sure.